app/views/overview/_list-row-actions.html

```diff
@@ -15,7 +15,11 @@
               <a href="" ng-click="row.startBuild(row.buildConfigs[0])">Start Build</a>
             </li>
             <li ng-if="'deploymentconfigs/instantiate' | canI : 'create'" role="menuitem">
-              <a href="" ng-click="row.startDeployment()">Deploy</a>
+              <a
+                href="" 
+                ng-attr-aria-disabled="{{canDeploy() ? undefined : 'true'}}"
+                ng-class="{ 'disabled-link': !canDeploy() }"
```

Comment:
Yeah, I think we've done that in other places. Note that "paused" isn't always the reason it's disabled, so if we add that message need to do it for other conditions